### PR TITLE
RELATED: RAIL-4258 Polish attribute filter examples

### PR DESCRIPTION
**Pull request by kandl**
- Split examples to tabs
- Add more customization examples
- Code cleanup in older examples

JIRA: RAIL-4258

---

Supported PR commands:

| Command                  | Description             |
| ------------------------ | ----------------------- |
| `ok to test`             | Re-run standard checks  |
| `extended test`          | BackstopJS tests        |
| `extended check sonar`   | SonarQube tests         |
| `extended check cypress` | Cypress E2E tests       |
| `extended check plugins` | Dashboard plugins tests |

---

# PR Checklist

-   [ ] commit messages adhere to the [commit message guidelines](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#what-should-the-commits-look-like)
-   [ ] review was done by a Code owner [if necessary](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#how-do-i-tell-if-my-pull-request-needs-approval-by-a-code-owner) (if you think it is not necessary, explain the reasoning in the description or in a comment)
-   [ ] `check` passes
-   [ ] `check-extended` passes
-   [ ] `check-extended-cypress` passes
-   [ ] `rush change` [was run if applicable](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#how-do-i-describe-my-changes-for-the-changelog)

### CODE REVIEW

File: examples/sdk-examples/src/examples/attributeFilter/index.tsx [hunk 1]
```diff
@@ -1,194 +1,194 @@
...
-        />
-        <h2>Attribute Filter customization</h2>
-        <ExampleWithSource
-            for={AttributeFilterCustomizationExample}
-            source={AttributeFilterCustomizationExampleSRC}
-            sourceJS={AttributeFilterCustomizationExampleSRCJS}
-        />
-    </div>
-);
+// // (C) 2007-2022 GoodData Corporation
```
Comment (no23reason) [inline examples/sdk-examples/src/examples/attributeFilter/index.tsx:1]: Should we remove this?

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/10089/ : SUCCESS in 17s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/12075/ : FAILURE in 8m 59s

File: examples/sdk-examples/src/examples/attributeFilter/customAttributeFilter/index.tsx [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+        <ExampleWithSource
+            for={AttributeFilterWithCustomizations}
+            source={AttributeFilterWithCustomizationsSRC}
+            sourceJS={AttributeFilterWithCustomizationsSRCJS}
+        />
+
+        <hr className="separator" />
+
+        <h2 id="use-attribute-filter-controller">useAttributeFilterController</h2>
+        <p>The easiest way to implement your own attribute filter is a useAttributeFilterController hook.</p>
```
Comment (no23reason) [inline examples/sdk-examples/src/examples/attributeFilter/customAttributeFilter/index.tsx:48]: is the hook

File: examples/sdk-examples/src/examples/attributeFilter/customAttributeFilter/index.tsx [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+            for={UseAttributeFilterHandler}
+            source={UseAttributeFilterHandlerSRC}
+            sourceJS={UseAttributeFilterHandlerSRCJS}
+        />
+
+        <hr className="separator" />
+
+        <h2 id="attribute-filter-handler">AttributeFilterHandler</h2>
+        <p>
+            AttributeFilterHandler comes up with a framework agnostic API, so you can integrate it with any
```
Comment (no23reason) [inline examples/sdk-examples/src/examples/attributeFilter/customAttributeFilter/index.tsx:78]: comes with (no up)

**Review by no23reason — APPROVED** [review]
LGTM

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by yenkins** [comment]
### Sonar scan result


More detail, see in https://sonarqube-gate.intgdc.com/dashboard?id=gooddata-ui-sdk-gate-PR3080

To scan for vulnerabilities in dependencies and run unit tests (to get coverage report in sonar) please comment your PR with 'extended check sonar'.

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-fossa-scanning-zuul-el7/2916/ : FAILURE in 2m 45s (non-voting)
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-merge-compliance-check/3222/ : SUCCESS in 21s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-sonarqube-scan-zuul-gate/2063/ : SUCCESS in 18m 47s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/12077/ : SUCCESS in 8m 27s

**Pull request merged**